.github/workflows/ci.yml

```diff
@@ -25,7 +37,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 18
+          node-version: ${{ inputs.print_tags || 18 }}
```

Comment:
```suggestion
          node-version: ${{ inputs.node_version }}
```

Reply:
Oh my. Thanks for catching that. :facepalm: